Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -231,15 +201,38 @@ final class ProjectEditorMapper: ProjectEditorMapping {
             defaultSettings: .recommended
         )
 
+        let helpersTarget: Target? = {
+            guard !helpers.isEmpty else { return nil }
+            return editorHelperTarget(
+                name: Constants.helpersDirectoryName,
+                filesGroup: manifestsFilesGroup,
+                targetSettings: targetWithLinkedPluginsSettings,
+                sourcePaths: helpers,
+                dependencies: editablePluginTargetDependencies
+            )
+        }()
+
+        let helperTargetDependencies = helpersTarget.map { [TargetDependency.target(name: $0.name)] } ?? []
+        let helperAndPluginDependencies = helperTargetDependencies + editablePluginTargetDependencies
+
+        let dependenciesTarget: Target? = {
+            guard let dependenciesPath = dependenciesPath else { return nil }
+            return editorHelperTarget(
+                name: "Dependencies",
+                filesGroup: manifestsFilesGroup,
+                targetSettings: targetWithLinkedPluginsSettings,
+                sourcePaths: [dependenciesPath],
+                dependencies: helperAndPluginDependencies
+            )
+        }()
```

Comment:
Fix: This makes it so remote helpers are linked as frameworks vs. a target in the editable project.

Also updates the dependencies target to depend on local helpers and plugins and link rmeote plugins